eth/downloader/queue.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 var (
-	blockCacheLimit   = 8192 // Maximum number of blocks to cache before throttling the download
-	maxInFlightStates = 4096 // Maximum number of state downloads to allow concurrently
+	blockCacheLimit   = 16384 // Maximum number of blocks to cache before throttling the download
```

Comment:
This is a lot of caching though and I'm afraid lower end devices will oom. I'd prefer to leave this as it was before.